app/Http/Resources/AdsResource.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class AdsResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
```

Comment:
probably `:array()` can be added.

Isn't it should be `public function toArray( Request $request)`?
I can see, in the comments it is the Request object.

Reply:
Unable to fix it, checked documentation but it's not working, can you please help.

https://laravel.com/api/5.8/Illuminate/Http/Resources/Json/JsonResource.html#method_toArray

![image](https://user-images.githubusercontent.com/99896518/155279692-530a6f15-aa19-45e6-88ed-08f7818f1972.png)

![image](https://user-images.githubusercontent.com/99896518/155281622-c9c98eb3-3fa0-4813-a071-c1395c7a9862.png)